tfac/dataHelpers.py

```diff
@@ -19,9 +20,22 @@ def importLINCSprotein():
     return pd.concat([dataA, dataB, dataC])
 
 
-def ohsu_data():
+def ohsu_data(export=False):
     """ Import OHSU data for PARAFAC2"""
-    return pd.read_csv(join(path_here, "tfac/data/ohsu/MDD_RNAseq_Level4.txt"), delimiter="\t")
+    RNAseq = pd.read_csv(join(path_here, "tfac/data/ohsu/MDD_RNAseq_Level4.txt"), delimiter=",", index_col=0)
```

Comment:
Why does the delimiter need to be changed? This is a specific, deposited data file provided by the LINCS consortium. There is value in working based on the file exactly as provided. I don't see why that's difficult if all that you changed was the delimiters.

Reply:
The original data was comma-delimited.

Reply:
There was no reason to make it a txt file either. So I just uploaded the data that existed in previous versions of the repo.

Reply:
Also I applied the 0.07 threshold for filtering genes in this repo.